action.yml

```diff
@@ -21,8 +25,19 @@ runs:
             sudo apt-get install "${packages[@]}"
           fi
         shell: bash
-      - name: "Compile documentation"
+      - name: "Compile documentation via makedoc.g"
+        if: ${{!inputs.use-makedoc-shell-script}}
```

Comment:
I don't think doing `if:` is allowed within a `runs.steps` item of a GitHub workflow (like this one) that's called from other workflow (as e.g. sonata will be).

But I can't get good information on this so far. I'm going to make an alternative PR that works, but which we might want to improve upon.